src/site/stages/build/process-cms-exports/transformers/node-health_care_region_page.js

```diff
@@ -359,11 +320,6 @@ module.exports = {
     'path',
     'field_govdelivery_id_emerg',
     'field_govdelivery_id_news',
-    'field_facebook',
-    'field_flickr',
-    'field_instagram',
-    'field_twitter',
-    'field_leadership',
```

Comment:
Why is this being removed?

Reply:
Leadership field is no longer in use on health_care_region_page in cms - has been removed, @jhonnyoddball cc: @kevwalsh